inc/class-command.php

```diff
@@ -158,15 +158,23 @@ protected function init( InputInterface $input, OutputInterface $output ) {
 			return;
 		}
 
-		$this->start( $input, $output );
+		$status = $this->start( $input, $output );
+		if ( $status !== 0 ) {
+			$output->writeln( '<info>Virtual machine could not be installed or launched.</>' );
+			return;
```

Comment:
Only thing to note here is that if the exit code is not 0 then you need to return it. Not returning anything just sets the default exit code of 0 which is why you see that pattern elsewhere in the method.